pkg/crc/api/handlers.go

```diff
@@ -16,6 +16,17 @@ import (
 	"github.com/code-ready/crc/pkg/crc/version"
 )
 
+func NewConfig() (crcConfig.Storage, error) {
+	viper, err := crcConfig.NewViperStorage(constants.ConfigPath, constants.CrcEnvPrefix)
+	if err != nil {
+		return nil, err
+	}
+	cfg := crcConfig.New(viper)
+	config.RegisterSettings(cfg)
+
+	return cfg, nil
+}
```

Comment:
Does this have to live in `pkg/crc/api/handlers.go`? `cmd/crc/cmd/root.go` has very similar code, maybe it's a good opportunity to share it?

Reply:
will have to change the return type to `config.Config` instead of `config.Storage`